src/ffi/client_manager.js

```diff
@@ -8,9 +8,26 @@ const _clientHandle = Symbol('clientHandle');
 
 class ClientManager extends FfiApi {
   static manifest = {
-    setNetworkListener: 'sync'
+    setNetworkListener: 'sync',
+    login: 'promise',
+    createAccount: 'promise'
   };
 
+  static _isUserCredentialsValid(locator, secret) {
```

Comment:
Can we have this as a private function? Or are you planning to expose this to the UI? If we are not exposing outside then this should be treated as private function